themes/default/content/templates/container-service/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Container Templates
+layout: overview
+description: Pulumi program templates are the fastest way to deploy [containerized applications](/containers) on [AWS](/aws), [Azure](/azure), or [Google Cloud Platform](/gcp). Templates come with predefined [infrastructure as code](/what-is/what-is-infrastructure-as-code) so you can get started instantly.
+meta_desc: Pulumi program templates that make it easy to deploy containerized applications on AWS, Azure, or Google Cloud Platform.
+meta_image: meta.png
+weight: 1
+---
+
+### What are containerized applications?
```

Comment:
Are we specifically talking about Containers or Containerized Applications. We should just make sure the Title and all these things line up with each other

Reply:
Yeah I think the problem here is we are focusing on applications and the whole point of these templates are really focused on bringing up the container service/cluster manager infrastructure